pkg/volume/cinder/attacher.go

```diff
@@ -185,7 +186,11 @@ func (attacher *cinderDiskAttacher) VolumesAreAttached(specs []*volume.Spec, nod
 	}
 
 	instanceID, err := attacher.nodeInstanceID(nodeName)
-	if err != nil {
+	if err == cloudprovider.InstanceNotFound {
+		glog.V(2).Infof("VolumesAreAttached: node %q does not exist.", nodeName)
```

Comment:
@rootfs if instannce is deleted, openstack cloud provider can't get its instanceID, so attacher.nodeInstanceID() return ("", err). Printing nodename is enough.
Yeah, it should use  glog.Warningf, thank you.